backend/.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+**/node_modules
```

Comment:
If possible, for the deleted `node_modules`, let's create a completely separate PR to remove them from the repo, so this PR can focus on the actual code changes.

So the steps should look like:
1. Open new PR to delete the node_modules folder
2. Submit and merge PR into your main branch
3. Rebase these open PRs onto your main branch (should be after the deletion of node_modules from github, but you should still see it in your local repo)

Reply:
Done